contracts/event/MultipleResultsEvent.sol

```diff
@@ -146,6 +154,8 @@ contract MultipleResultsEvent is NRC223Receiver, Ownable {
         _resultSetStartTime = resultSetStartTime;
         _resultSetEndTime = resultSetEndTime;
         _centralizedOracle = centralizedOracle;
+        _arbitrationLength = arbitrationLength;
+        _arbitrationRewardPercentage = arbitrationRewardPercentage;
```

Comment:
can bad user pass in super short arbitration length and arbitrary arbitration reward percentage? I think we need to validate these values

Reply:
its validated in L170 `config.calculateThreshold()`. if the arbitrationLength does not match the set values then it will return 0 and throw an error.

as for the arbitration reward, how would the contract know what is too big (other than being less that 100, which is an impossible percentage)? already have validation on L145 for it to be less that 100%. think the percentage validation should happen at the UI level. if we keep changing it here we would have to deploy a whole new EventFactory version just to change the percentage.